sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/DownloadCBDE.targets

```diff
@@ -74,20 +64,12 @@
     <Delete Files="@(ExistingCBDEFiles)" Condition="$(ShouldDownloadCBDE)=='true'"/>
 
     <Message Importance="high" Text="CBDE: Begin downloading CBDE zip at @(CBDEUrls)..." Condition="$(ShouldDownloadCBDE)=='true'"/>
-    <Exec ConsoleToMsBuild="true" Condition="$(ShouldDownloadCBDE)=='true'"
-      Command="$(PowerShellExe) -NonInteractive -ExecutionPolicy Unrestricted -File $(DownloadScript) -url @(CBDEUrls) -outfolder $(CBDECurrentFolder) -outfile %(CBDEUrls.ZipFileName)">
```

Comment:
you should also remove the `UnzipScript` and `DownloadScript` (lines 45-46) and remove the `unzip.ps1` and `download.ps1` scripts from the disk, right?

Reply:
Right, I just forgot :)